amazon-kinesis-client/src/main/java/software/amazon/kinesis/coordinator/Scheduler.java

```diff
@@ -89,6 +98,8 @@
 @Slf4j
 public class Scheduler implements Runnable {
 
+    private static final long NEW_STREAM_CHECK_INTERVAL_MILLIS = 10000L;
```

Comment:
Should this be in the app initialization config ? This seems so frequent. Archiver as of now sync for every 5mins which is longer. I think we should move this to config and have some default values like 1 min or so. Thoughts ?